packages/apps-config/src/api/params/proposalThresholds.ts

```diff
@@ -1,38 +1,31 @@
 // Copyright 2017-2021 @polkadot/app-config authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import type { ApiPromise } from '@polkadot/api';
-
 import { KULUPU_GENESIS, KUSAMA_GENESIS, POLKADOT_GENESIS } from '../constants';
 
-const PROPOSE_THRESHOLDS: Record<string, number> = {
+export const PROPOSE_THRESHOLDS: Record<string, number> = {
```

Comment:
Here Kulupu doesn't seem correct.

@sorpaas This should be 0.8, 4/5 - https://github.com/kulupu/kulupu/blob/master/runtime/src/lib.rs#L602 ?